fire/SFMainMod.F90

```diff
@@ -1218,16 +1218,19 @@ subroutine  active_crown_fire ( currentSite)
                 ! calculate canopy fraction burnt, Eq 28 Scott & Reinhardt App A 
                 ! using wind at which ROS_active = ROS_active_min
                 ! solve for phi_wind associated with ROS_active_min
-                phi_wind_ROS_SA = (ROS_active_min * fuel_bd * eps * q_ig - 3.34 * ir * xi) &
-                                     /(3.34 * ir * xi)
+                if (ir > 0._r8 .and. xi > 0._r8) then
+                   phi_wind_ROS_SA = (ROS_active_min * fuel_bd * eps * q_ig - 3.34 * ir * xi) / (3.34 * ir * xi)
+                else
+                   phi_wind_ROS_SA = 0._r8
+                end if
```

Comment:
@jkshuman running this by you:
I was getting `phi_wind_ROS_SA = infinity` due to `ir = 0`, which was due to `moist_damp = 0`. I added this if statement to confirm that `ir > 0 .and. xi > 0`.
**Question 1:** Did I make the right choice for the `else`?

This change caused `canopy_frac_burnt < 0` a few lines down due to `ROS_SA = 0`. So I added `max(0...` to the calculation of `canopy_frac_burnt`.
**Question 2:** Does that seem reasonable?

I have now completed a 5 yr simulation at the CZ2 site. I will share some figures here on github to initiate discussion.

Reply:
Just saw your message about other updates @jkshuman 
I will try those out before posting figures. Thanks.

Reply:
@jkshuman 
in the context of your latest modifications, I'm keeping the if statement and the max(0... that I talked about above, because they still seem relevant when moist_damp = 0 and ROS_SA = 0.

Reply:
@slevisconsulting normally I would say yes, sounds good, but there must be something more going on in here. you are in this section because the patchFI >= passive_crown_FI indicating there is a fire. And then we are calculating the pieces to get the reaction intensity "ir" for the active crown fire using a given fuel loading for the FM 10 model.  It would seem that I have a mistake in this section prior to calculating "ir". I am looking it over now. I belive it is a conversion error. Let me make an adjustment.